server_admin/topics/roles-groups/con-role-scope-mappings.adoc

```diff
@@ -13,7 +13,7 @@ By default, each client gets all the role mappings of the user. You can view the
 .Full scope
 image:{project_images}/full-client-scope.png[Full scope]
 
-By default, the effective roles of scopes are every declared role in the realm. To change this default behavior, toggle *Full Scope Allowed* to *ON* and declare the specific roles you want in each client.  You can also use <<_client_scopes, client scopes>> to define the same role scope mappings for a set of clients.
+By default, the effective roles of scopes are every declared role in the realm. To change this default behavior, set the *Full Scope Allowed* option to *OFF* and declare the specific roles you want in each client.  You can also use <<_client_scopes, client scopes>> to define the same role scope mappings for a set of clients.
```

Comment:
@little-pinecone We are standardizing on this format for something you enable: “Toggle *<switch-name>* to *ON*.”

Reply:
@andymunro In this particular case we want to disable the default option to be able to declare roles manually. Therefore, I changed it to "_toggle *Full Scope Allowed* to *OFF*_". Is my understanding correct?